router/pkg/cors/cors.go

```diff
@@ -123,21 +123,27 @@ func (c *Config) parseWildcardRules() [][]string {
 			continue
 		}
 
-		if c := strings.Count(o, "*"); c > 1 {
```

Comment:
Please provide a bench for the regex so we can validate before and after impact.

Reply:
added two benches